doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -0,0 +1,169 @@
+
+//Old wording - remove once new is OK. For a StatefulSet with `N` replicas, when pods are being deployed, they are created sequentially, in order from {0..N-1}.
+
+
+ifdef::miq[]
+[[building-images]]
+=== Building Images on OpenShift
+
+You can build the images from this repository using OpenShift:
+----
+$ oc -n <your-project> new-build --context-dir=images/cfme-app https://github.com/CloudForms/cloudforms-pods#master
+----
+
+Additionally, Red Hat recommends setting the following `dockerStrategy` parameters to ensure a fresh build every time:
+----
+$ oc edit bc -n <your-project> cloudforms-pods
+
+strategy:
+  dockerStrategy:
+    forcePull: true
+    noCache: true
+----
+To execute a new build after the first (automatically started) build, run:
+----
+$ oc start-build -n <your-project> cloudforms-pods
+----
+Configure the following template parameters on the newly built image:
+----
+$ oc new-app --template=cloudforms \
+  -n <your-project> \
+  -p APPLICATION_IMG_NAME=<your-docker-registry>:5000/<your-project>/cloudforms-pods \
+  -p APPLICATION_IMG_TAG=latest \
+  ...
+----
+
+endif::miq[]
+
+
+[[creating-backups]]
+=== Creating a Backup
+
+// 35:10
+
+Create a persistent volume for backups using the PV backup template (`cfme-pv-backup-example.yaml`) in case you need to restore to a previous version.
+
+. Create the persistent volume for the backup:
++
+----
+$ oc create -f cfme-pv-backup-example.yaml
+----
++
+. Create the backup persistent volume claim (PVC):
++
+----
+$ oc create -f cfme-backup-pvc.yaml
+----
++
+. Verify the persistent volume claim was created:
++
+----
+$ oc get pvc
+----
++
+. Back up secrets, such as database encryption keys and credentials.
++
+[IMPORTANT]
+====
+Be careful to back up secrets in a secure location.
+====
++
+----
+$ oc get secret -o yaml --export=true > secrets.yaml
+$ oc get pvc -o yaml --export=true > pvc.yaml
+----
++
+. Initiate the database backup:
++
+----
+$ oc create -f cfme-backup-job.yaml
+----
+
+This step creates a container, connects to the database pod, pg_basebackup. ?
+
+[[backup-restore]]
+=== Restoring to a Backup
+
+You can restore to a database backup created in xref:creating-backups[] using the restore template (`cfme-restore-job.yaml`).
+
+The restore job will look for `cfme-backup` and `cfme-postgresql` PVs by default, and the latest successful backup will be restored by default. If existing data is found on prod PG volume, it will be renamed and left on the volume.
+
+[IMPORTANT]
+====
+You must perform a database restore on an offline environment. All pods must be scaled down to 0, and not running.
+====
+
+. Shut down all pods:
++
+----
+$ oc stop all pods
+----
++
+. To initiate the database restore, create the restore template:
++
+----
+$ oc create -f cfme-restore-job.yaml
+----
+
+After [INSERT RESULT], you can scale pods back up.
```

Comment:
Is [INSERT RESULTS] intentional? if this is a placeholder of I'm misunderstanding for lack of my own personal knowledge please ignore this comment.